action/library/src/main/java/io/knotx/fragments/action/library/cache/operations/CacheActionLogger.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2019 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.knotx.fragments.action.library.cache.operations;
+
+import static java.time.Instant.now;
+
+import io.knotx.commons.time.TimeCalculator;
+import io.knotx.fragments.action.api.log.ActionLogLevel;
+import io.knotx.fragments.action.api.log.ActionLogger;
+import io.knotx.fragments.api.FragmentResult;
+import io.vertx.core.json.JsonObject;
+
+public class CacheActionLogger {
```

Comment:
Wrapped `ActionLogger` in order to extract responsibility of log format and level to a separate class.